ajax/libs/html5-device-mockups/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "html5-device-mockups",
+  "filename": "device-mockups.min.css",
+  "description": "HTML5 mockups of popular devices, to showcase your portfolio and spice up your website.",
+  "keywords": [
+    "html5",
+    "device",
+    "mockups",
+    "demo",
+    "portfolio"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pixelsign/html5-device-mockups.git"
+  },
+  "homepage": "https://pixelsign.github.io/html5-device-mockups/",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/pixelsign/html5-device-mockups.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "../device-mockups/*/!(*.psd)",
```

Comment:
We don't use `..` please remove this line.
Also this pattern can't get any files under version folder.

Please add another rule, the whole auto-update config is:
```js
  "autoupdate": {
    "source": "git",
    "target": "https://github.com/pixelsign/html5-device-mockups.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/*.css"
        ]
      },
      {
        "basePath": "",
        "files": [
          "device-mockups/**/!(*.psd)"
        ]
      }
    ]
  },
```

Thanks

Reply:
@sufuf3 But with that rule, the css files can not fetch the images they need.
e.g.
In https://github.com/pixelsign/html5-device-mockups/blob/master/dist/device-mockups.css#L1025, it uses ```../device-mockups/Chromebook/portrait_black.png``` as its background image path. However, with this rule, the image will be ```device-mockups/Chromebook/portrait_black.png```, which is different.

Reply:
Got it.
Because the file structure of this lib is as the following. (`dist` and `device-mockups` folder should be under the same layer)
![2018-01-04 10 24 04](https://user-images.githubusercontent.com/8349587/34547910-706b0ad8-f139-11e7-957e-495129246813.png)

Then, maybe we can use the pattern as the followings, what do you think?
```js
      {
        "basePath": "",
        "files": [
          "dist/**/*",
          "device-mockups/**/!(*.psd)"
        ]
      }
```

Because the device-mockups folder would not under lib's folder when using the current pattern. It's a little bit weird.
PS. also need to update the filename field to `dist/device-mockups.min.css`

Many thanks.